cassiopeia/data/CassiopeiaTree.py

```diff
@@ -1872,6 +1872,105 @@ def get_distances(
             for _node in (self.leaves if leaves_only else self.nodes)
         }
 
+    def scale_to_unit_length(self) -> None:
+        """
+        Scales the tree to have unit length.
+
+        The longest path from root to leaf will have length 1 after the
+        scaling.
+
+        Raises:
+            CassiopeiaTreeError if the tree has not been initialized.
+        """
+        self.__check_network_initialized()
+
+        current_times = self.get_times().values()
```

Comment:
Is there ever a case when the values are not initialized?

Reply:
Only if the network is not initialized, which is checked for in the previous line, or am I missing something?

Reply:
yeah you're right -- if the network is initialized through `populate_tree`, then this should be just fine.